data/remnant/remnant 2 cognizance.txt

```diff
@@ -116,23 +114,22 @@ mission "Remnant: Cognizance 2"
 		personality forbearing timid uninterested surveillance staying
 		fleet
 			names "korath"
-			cargo 1
-			commodities "void sprite parts"
+			cargo 3
```

Comment:
Likewise here: There *is* a "void sprite parts" commodity listed in `commodities.txt`, and this is supposed to pull from that list. If that's not working, I'd like to fix it rather than replace that cargo with random miscellaneous stuff.